### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by amihos** [FranzKafkaYu/x-ui#158] [Ideas]
Can you add API for adding, removing, editing inbounds?

**Comment by am-fallah** [comment]
@amihos 
Did you find out how to use api in this panel ?

**Comment by Hoseinkhalaj1** [comment]
I need API too

**Comment by zamanimatin** [comment]
@FranzKafkaYu 
Can you help us with some suggestions in order to use some api for some inbounds modifications?
